### ISSUE TITLE
False sensor trigger

### ISSUE THREAD

**Issue by marceldeklerk** [kukulich/home-assistant-jablotron100#70]
I did an interesting observation this morning. My alarm was armed entire night (between 11:18pm yesterday and 7:19am today).
![image](https://user-images.githubusercontent.com/25645691/225842944-6e9c7b96-0789-4ba1-8bda-3b37ddf38cd1.png)

Though I noticed that Homeassistant registered a triggering of my doorsensor at 6:48am
![image](https://user-images.githubusercontent.com/25645691/225843223-284f271c-040c-4b4b-a029-c587ada48a8e.png)

At this time, the Jablotron system was armed, so that trigger must have been a false one, else it definitely would have tripped the alarm, which it didn't. Just to be sure, I checked the security camera recordings of by back-yard, but as expected, nobody opened the door of course. 

Not sure if this is a glitch that I caught by accident, or if there is something wrong in the interpretation logic (so a certain package send by the Jablotron system is misinterpreted by the integration as sensor trigger). 

Posting here to check if anyone else has experienced this. I will keep an close look and check every morning from now if I can reproduce the issue.

**Comment by kukulich** [comment]
@marceldeklerk Please enable logs for all devices. We may find something useful there.

**Comment by marceldeklerk** [comment]
Sure done. Not sure if I will be able to reproduce though. Lets see.
I will create an automation to send me a push notification to my phone whenever a door sensor is triggered while the system is armed.

**Comment by marceldeklerk** [comment]
Ok, this morning it happened again. More or less around the same time as yesterday. I have two wireless doorsensors, as far as I can tell so far, it only happens to my backdoor sensor.

![image](https://user-images.githubusercontent.com/25645691/226088547-5afd3a8d-7111-4c35-8c76-1496270571a0.png)
Yesterday: 6:48.50am
Today: 6:59.13am

The Jablotron100 component log at time of false triggering:

2023-03-18 06:59:10.048 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-18 06:59:10.550 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-18 06:59:11.053 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-18 06:59:11.340 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 7): 9005079c02091a
2023-03-18 06:59:12.842 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-18 06:59:13.345 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-18 06:59:13.518 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 7): 55094f84c00180ec660f09
2023-03-18 06:59:15.023 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-18 06:59:15.526 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-18 06:59:16.028 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102

**Comment by kukulich** [comment]
@marceldeklerk Can you please send me the info, how is the sensor set in Jablotron?

- Is is triggering alarm when armed partially or only when armed completly?
- Does it have some delay before triggering alarm?
- Is it set as A, B or C alarm type? Or is it set differently?

**Comment by marceldeklerk** [comment]
- It is triggering when the alarm is set partially (armed_home). Note that in in armed_home mode my doorsensors will cause an alarm when triggered, the motionsensors are not used in armed_home mode. I haven't been able to try if the same happens in armed_away mode. For tonight I will leave the Alarm off (Disarmed) to see if the integration registers a triggereing of the backdoor sensor tomorrow morning (around 7am) again.

- When the doorsensor is triggered, there is a 20 second delay before the alarm is triggered. Just to be clear here: The false sensor triggering was when the alarm was set in armed_home mode, the Jablotron100 Home Assistant integration captured a tripping of the doorsensor. The Jablotron Alarm itself did not capture this triggering (because the door was actually closed), so i expect that the integration misinterprets a package sent as sensor trigger.

- I don't understand your last question. Can you elobarate?

**Comment by marceldeklerk** [comment]
It is the next day now. I left the Jablotron alarm **disarmed** last night, but false triggering happened again. I think it is safe to assume that the false triggering of the sensor happens, regardless of the arming state of the alarm system. 

![image](https://user-images.githubusercontent.com/25645691/226160296-25a656d3-2fe5-46f5-b99b-44a10f8b3cf4.png)

**False triggers recorded so far:**
Friday: 6:48:15am
Saturday: 6:59:13am
Sunday: 7:10:16am
Looks like there is a pattern. Every false trigger is **24 hours and 11 minutes after the previous**. Based on previous false triggers I expect that tomorrow (Monday) the false trigger will be at 7:21am.

Note that I have two wireless door sensors of the same model, I wonder if the same issue happens with the other wireless sensor as well.... At night it is easy to identify, but during daytime that its a lot more difficult, I will keep an close eye on it.

**Comment by kukulich** [comment]
![Clipboard01](https://user-images.githubusercontent.com/260445/226194643-9a3a18f3-33c9-442e-9379-d32d11f04df5.png)

A/B/C etc are timer settings - if the device has some delay it should one of this types.

**Comment by kukulich** [comment]
Please try to install version `master` in HACS. I've just added possible fix. However, please send me packets for device 7 when the door are really open - I would like to check the difference between relevant packets and the false positive packets.

**Comment by marceldeklerk** [comment]
I just installed the master version of the Jablotron100 integration via HACS.

Below are the packets for when the door (sensor 7) is really open and closed

2023-03-19 19:56:10.562 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:11.064 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:11.416 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 7): 55090184c00140e0f3150c
2023-03-19 19:56:12.920 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:13.026 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080080000000000000
2023-03-19 19:56:14.532 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:15.033 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:15.535 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:16.037 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:16.539 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:17.042 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:17.412 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 7): 55090186c001a0e013160c
2023-03-19 19:56:18.915 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-19 19:56:19.418 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102

Below the A/B/C timer settings, do you still think this is relevant?
![image](https://user-images.githubusercontent.com/25645691/226201087-ad09acb2-c9ce-4ae3-9d1b-29f7f12b8c27.png)

**Comment by kukulich** [comment]
> When the doorsensor is triggered, there is a 20 second delay before the alarm is triggered

Is it really 20 seconds? I think it should be 15 or 30 or 60 - based on the A/B/C settings...

**Comment by kukulich** [comment]
You can also open "Control panel" -> "Events" -> "Online signals". If you open the door, there should be new event and in "Event" column there should be info about the settings.

**Comment by marceldeklerk** [comment]
> > When the doorsensor is triggered, there is a 20 second delay before the alarm is triggered
> 
> Is it really 20 seconds? I think it should be 15 or 30 or 60 - based on the A/B/C settings...

I always assumed it was around 20 seconds, but based on checking the settings in the Jablotron F-Link software, I see it indeed is 15 seconds.

**Comment by marceldeklerk** [comment]
> You can also open "Control panel" -> "Events" -> "Online signals". If you open the door, there should be new event and in "Event" column there should be info about the settings.

See below:
![image](https://user-images.githubusercontent.com/25645691/226206883-79e93b8e-f180-45ea-a9ed-f4d450dab551.png)

Not really sure what you are looking for, as this does not give a lot of information I think?
Note that I had to walk from kitchen to livingroom to reach the door with sensor 7, hence the triggering of the other 2 devices (4 + 5) as well.

**Comment by marceldeklerk** [comment]
It didn't trigger last night. Can you explain what was the issue, and do you understand why this only happens to one of my two wireless door sensors? I will keep it under observation for a bit longer.
Thanks anyways!

**Comment by kukulich** [comment]
@marceldeklerk This integration is based on reverse engineering. Device states are still not completly understood. I expect that keepalive packet was wrongly detected as "state" packet. However it's just an assumption.

> do you understand why this only happens to one of my two wireless door sensors

The state packets are different based on the device settings - that's why I was interested how the device is set.

If you can sabotage your door sensor (open it?), it would be interested how the packets for the sabotage and also for the end of the sabotage (closing it?) look.

**Comment by marceldeklerk** [comment]
Sure. I can try that. Probably will do that towards the end of the week.

**Comment by marceldeklerk** [comment]
This is creating a sabotage (opening doorsensor 6) while the alarm is disarmed:

2023-03-25 14:47:18.841 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:19.343 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 80080339393935343136
2023-03-25 14:47:19.343 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 52021305
2023-03-25 14:47:19.356 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 800d0cffffffff0100018009200000
2023-03-25 14:47:19.356 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 5203820113
2023-03-25 14:47:19.857 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0089668823f0fbd8e11
2023-03-25 14:47:20.358 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55090680800100bd2e2712
2023-03-25 14:47:20.860 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0081a80800100bdae11
2023-03-25 14:47:21.361 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0081780800100bdce11
2023-03-25 14:47:21.862 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 512201110700070007000700070007000700070007000700070007000700070007000010
2023-03-25 14:47:22.364 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 9005069c02081a
2023-03-25 14:47:22.866 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d2068d5e55000000
2023-03-25 14:47:23.367 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55090180800130bd4e2714
2023-03-25 14:47:23.869 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010600
2023-03-25 14:47:24.370 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080040020000000000
2023-03-25 14:47:25.862 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:26.364 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:26.866 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:27.368 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:27.870 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:28.371 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:28.535 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010300
2023-03-25 14:47:30.036 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:30.538 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:31.040 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:31.542 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:32.044 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:32.546 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:33.048 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:33.550 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:34.052 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:34.323 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080040020000000000
2023-03-25 14:47:35.825 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:36.284 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55090182800100be6e2711
2023-03-25 14:47:37.792 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:38.294 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:38.796 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:39.298 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:39.799 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:40.300 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:40.802 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:41.304 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:41.494 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55090682800160be8e2709
2023-03-25 14:47:41.997 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0081b80800160beee11
2023-03-25 14:47:42.498 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 9005069c020719
2023-03-25 14:47:43.998 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:44.200 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55094f80800180beae2709
2023-03-25 14:47:45.320 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080000020000000000
2023-03-25 14:47:46.822 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:47.323 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:47.824 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:48.326 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:48.828 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:48.847 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d00818708200d0be0e12
2023-03-25 14:47:49.348 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d00896708200dfbe2e12
2023-03-25 14:47:49.849 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 512201010700070007000700070007000700070007000700070007000700070007000010
2023-03-25 14:47:51.350 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:51.852 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:47:52.190 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d008967082000fbf4e12
2023-03-25 14:47:52.691 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500
2023-03-25 14:47:53.192 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500
2023-03-25 14:47:53.712 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500
2023-03-25 14:47:54.715 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500
2023-03-25 14:47:55.715 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500
2023-03-25 14:47:56.216 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d008967082004fbf6e12
2023-03-25 14:47:56.717 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080000020000000000
2023-03-25 14:47:57.218 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500

**Comment by marceldeklerk** [comment]
The same as above, but now in armed_home mode. This time I managed the close the doorsensor housing (sensor 6) a lot faster compared to the first time:

2023-03-25 14:53:46.389 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:53:46.891 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:53:47.392 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:53:47.894 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:53:48.396 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:53:48.898 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:53:49.336 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080000020000000000
2023-03-25 14:53:50.006 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 10): 900c0a9c0907278316001f213300
2023-03-25 14:53:50.508 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 550906808001f0d64e280c
2023-03-25 14:53:51.011 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0081a808001f0d68e15
2023-03-25 14:53:51.512 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 512202110700070007000700070007000700070007000700070007000700070007000010
2023-03-25 14:53:52.014 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d00817808001f0d6ae15
2023-03-25 14:53:52.515 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 9005069c020618
2023-03-25 14:53:53.776 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55090180800120d76e2814
2023-03-25 14:53:54.279 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0081580800120d7ce15
2023-03-25 14:53:54.780 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 512242110700070007000700070007000700070007000700070007000700070007000010
2023-03-25 14:53:55.282 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d206ac5e55000000
2023-03-25 14:53:55.783 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55090182800140d78e2814
2023-03-25 14:53:56.285 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0081680800140d7ee15
2023-03-25 14:53:56.786 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010600
2023-03-25 14:53:57.287 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55090682800150d7ae2811
2023-03-25 14:53:57.791 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0081b80800150d70e16
2023-03-25 14:53:58.802 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 9005069c020718
2023-03-25 14:54:00.305 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:54:00.322 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 512242110700070007000700070007000700070007000700070007000700070007000010
2023-03-25 14:54:00.823 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d0089c68803f80d74e16
2023-03-25 14:54:01.324 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080040020000000000
2023-03-25 14:54:01.826 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 6): 55094f80800190d7ce2811
2023-03-25 14:54:02.327 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d00818688200a0d76e16
2023-03-25 14:54:02.828 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming (device 2): 55080c698200a0d7ee28
2023-03-25 14:54:02.828 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Active user: 0
2023-03-25 14:54:03.329 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d00896688200afd78e16
2023-03-25 14:54:03.830 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010300
2023-03-25 14:54:04.331 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 512201010700070007000700070007000700070007000700070007000700070007000010
2023-03-25 14:54:04.833 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d008c38c0000a0d7ae16
2023-03-25 14:54:05.334 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d00829688200a0d7ce16
2023-03-25 14:54:05.835 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d008966882002fd8ee16
2023-03-25 14:54:06.336 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 512201000700070007000700070007000700070007000700070007000700070007000010
2023-03-25 14:54:07.829 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:54:08.332 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:54:08.834 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:54:09.336 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:54:09.838 DEBUG (ThreadPoolExecutor-2_1) [custom_components.jablotron100] Outcoming: 520102
2023-03-25 14:54:09.845 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500
2023-03-25 14:54:10.725 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500
2023-03-25 14:54:11.331 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: d8080000020000000000
2023-03-25 14:54:11.833 DEBUG (ThreadPoolExecutor-2_0) [custom_components.jablotron100] Incoming: 44057f39010500

Are you interested in these packages because you are trying to trip the alarm from home assistant by simulating a sabotage?

**Comment by kukulich** [comment]
@marceldeklerk I'm sorry but I was expectinh the sabotage of the device 7 because that's the sensor that reported false positives :)

**Comment by kukulich** [comment]
> Are you interested in these packages because you are trying to trip the alarm from home assistant by simulating a sabotage?

No, I'm just trying to understand device packets better so the integration can report the states more right.

**Comment by marceldeklerk** [comment]
Ok. Will generate for sensor 7 tomorrow.

**Comment by marceldeklerk** [comment]
I just tried to trigger a sabotage for door sensor 7, and guess what.... I am unable to so. The Jablotron alarm did not register it at all. It just detected a normal trigger of the sensor (because when opening the housing, I moved it away from the magnet), but the system did not register a sabotage as it did with sensor 6.
I checked the configuration in F-Link and compared the settings of both door sensors (6 and 7), but was not able to see any difference between the two. I think the that the small sabotage switch in sensor 7 might be broken or something? This might also explain the fact that only sensor 7 was sending the message every 24 hours that your integration interpreted as a normal trigger. Maybe the package means an issue with the sabotage switch?
